scripts/fvt-tools.sh

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bash
+
+# Wait loop configuration
+SLEEP_COUNT=30
+SLEEP_WAIT_SECONDS=2
+declare -a NODES
+
+fvtlog() {
+  echo "$(date +"%F %T %Z")" "[fvt]" "$*"
+}
+
+list_nodes() {
```

Comment:
This function doesn't lists nodes, it fetch nodeas and sets a global variable. My suggestion looks like this:
```
list_nodes() {
    kubectl get nodes --selector "node-role.kubernetes.io/master notin ()" --no-headers -o jsonpath='{.items[*].metadata.name}'
}
```
and
```
NODES=($(list-nodes))
```

Reply:
Cool, good idea! Thank you!